### PR TITLE
Update smoke tests

### PR DESCRIPTION
**Pull request by coverbeck**
**Description**
Get smoke tests working.

1. On prod and staging, we would sometimes run into the search self-DOS. This would cause the test to fail and our WAF rate limit to be hit. Removed the test that led to that by creating staging and prod-specific files that comment out the test that causes the self-DOS. Created a followup ticket, [SEAB-4767](https://ucsc-cgl.atlassian.net/browse/SEAB-4767), to restore the tests once 1.13 is deployed to staging/prod.
2. In looking at this, noticed we were installing chrome and chromedriver twice (they're installed in `setup_nightly_tests`, so removed.
3. Another issue was introduced by the GitHub app logs fix -- before, when you deleted the user's last tool, the `There are currently no tools registered under this account` message would display, now it doesn't. Created an issue to follow up on that, dockstore/dockstore#5105.

Smoke tests passed although the staging auth one is still flaky :(.

**Issue**
dockstore/dockstore#4588

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain

### CODE REVIEW

File: cypress/integration/smokeTests/prodOnly/search.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      cy.url().should('not.include', '/search?descriptorType=WDL');
+      cy.contains('the Language is WDL').should('not.exist');
+      cy.go('forward');
+      cy.url().should('include', '/search?descriptorType=WDL');
+      cy.contains('the Language is WDL').should('exist');
+    });
+    it('should be able to use basic search box and have suggestions', () => {
+      cy.get('[data-cy=basic-search]').type('dockstore_i{enter}');
+      cy.contains(' Sorry, no matches found for dockstore_i');
+      cy.contains(/Do[ ]you[ ]mean:[ ].+/);
```
Comment (svonworl) [inline cypress/integration/smokeTests/prodOnly/search.ts:34]: The code robot has a valid beef about these, I think.  It says we should replace the character class with the actual character, for example:
```
cy.contains(/Do you mean: .+/);
```

**Review by svonworl — APPROVED** [review]

File: cypress/integration/smokeTests/prodOnly/search.ts (hunk 1 above)
Comment (coverbeck) [reply cypress/integration/smokeTests/prodOnly/search.ts:34]: It's not clear from my PR description, but I took "cypress/integration/smokeTests/**sharedTests**/search.ts", copied it to "cypress/integration/smokeTests/**stagingOnly**/search.ts" and "cypress/integration/smokeTests/**prodOnly**/search.ts", then commented out lines 43-55 in the copied versions. Lines 43-55 are what are causing the self-DOSsing.

This is the pattern for running dev/staging/prod specific tests.

Once 1.13 is deployed I will delete this file and the `stagingOnly` copy as well.

TLDR; it's a copy of existing code, and the copy is going to get deleted, so I don't think it's worth it for this PR. But I'll create a followup ticket for the "core" test.

Comment (coverbeck) [reply cypress/integration/smokeTests/prodOnly/search.ts:34]: On further reflection, I think I'll pass on creating a ticket. The next person who edits the core file can deal with it then, but it's working for now. Feel free to push back.

**Review by denis-yuen — APPROVED** [review]

**Review by kathy-t — APPROVED** [review]
